### DISCUSSION TITLE
[QUESTION] bot still leaves the vc after defining leaveonend and leaveonempty

### DISCUSSION THREAD

**Discussion by astrex1969** [Androz2091/discord-player#618] [Q&A]
**Question**

Ee idk why but i have disabled leaveonEmpty and leaveonEnd 
I use autoplay to continuously play music but the bot disconnects after 30-45 mins if no ones in the channel.

**Comment by twlite** [comment]
Few weeks ago, I left my bot on autoplay mode overnight and it was fine 🤔

I guess some stream error stopped the player?

**Comment by astrex1969** [comment]
> Few weeks ago, I left my bot on autoplay mode overnight and it was fine 🤔
> 
> I guess some stream error stopped the player?

Ig not cuz I tried it with other songs and even after restarting it wasn't working . 
This is how I have defined player client
`
const player = new Player(client, {            enableLive: true,            leaveOnEmpty: false,            leaveOnEnd: false,            leaveOnStop: false,            autoSelfDeaf: false,           });`
<br class="Apple-interchange-newline">
